<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLens</class>
 <widget class="QDialog" name="AddLens">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 117, 143);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbllensiol">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens IOL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbllensdetails">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Details</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineiol">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenstype">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <item>
    <property name="text">
     <string>Anterior</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Posterior</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbllensasrkt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ASRKT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineASRKT">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllasrkii">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ASRKII</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineASRKII">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllacd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ACD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineACD">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllsf">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>SF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSF">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblla0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>a0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linea0">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblla1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>a1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linea1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblla2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>a2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linea2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButLensSave">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>610</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButLensBack">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>610</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
